modules/agar/src/main/ts/ephox/agar/arbitrary/ArbChildrenSchema.ts

```diff
@@ -37,10 +37,10 @@ const composite = (rawDepth, detail, construct) => {
 const structure = (rawDepth, detail, construct) => {
   const components = toComponents(detail);
   return Jsc.number(0, 1).generator.flatMap((random) => {
-    const children = Arr.foldl(components, (b, component) => {
+    const children = Arr.foldl(components, (b, component) =>
       // TODO: Allow the order to be mixed up?
-      return random <= component.chance ? b.concat([ construct(component.component, rawDepth) ]) : b;
-    }, []);
+      random <= component.chance ? b.concat([ construct(component.component, rawDepth) ]) : b
```

Comment:
This looks pretty messy/hard to read... This is possibly a case where it should remain as a multiline function?

Reply:
I tried to improve it. I don't find it too hard to read but it's not ideal.

Reply:
I could make it a multi-line function by assigning the test to a constant or I could use an if statement instead of a ternary.

Reply:
It's more it's visually jarring to have the comma floating out in the open like it is, as it makes it harder to follow, so I'd opt for the if statement personally 🤷‍♀

Reply:
you mean the comma between the function and the array in the `foldl` call, which is now indented quite a bit? It's a bit weird, but I'm getting used to that sort of thing more recently.

Reply:
It's changed now, so it's a bit harder to describe, but previously you had something like:
```
const a = Arr.foldl(arr, (acc, b) =>
  b
, [])
```
so the comma/array is floating in the middle of nowhere.

Reply:
Honestly while this rule is generally useful, I'm finding a number of issues with the formatting it's done. Some can be fixed, but things like this (eg foldl/foldr) don't seem like a good place to be removing the braces.

Reply:
ah ok, yeah I think the auto-fix for eslint is just brute force without any opinions on the resulting code format